x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/analysis/Verifier.java

```diff
@@ -657,4 +659,31 @@ private static void checkMatchCommand(LogicalPlan plan, Set<Failure> failures) {
             }
         }
     }
+
+    private static void checkFullTextQueryFunctions(LogicalPlan plan, Set<Failure> failures) {
+        if (plan instanceof Filter f) {
+            Expression condition = f.condition();
+            if (condition instanceof FullTextFunction ftf) {
+                // Similar to cases present in org.elasticsearch.xpack.esql.optimizer.rules.PushDownAndCombineFilters -
+                // we can't check if it can be pushed down as we don't have yet information about the fields present in the
+                // StringQueryPredicate
+                plan.forEachDown(LogicalPlan.class, lp -> {
+                    if ((lp instanceof Filter || lp instanceof OrderBy || lp instanceof EsRelation) == false) {
+                        failures.add(
+                            fail(
+                                plan,
+                                "{} function cannot be used after {}",
+                                ftf.functionName(),
+                                lp.sourceText().split(" ")[0].toUpperCase(Locale.ROOT)
+                            )
+                        );
+                    }
+                });
+            }
+        } else {
+            plan.forEachExpression(FullTextFunction.class, ftf -> {
+                failures.add(fail(ftf, "{} function is only supported in WHERE commands", ftf.functionName()));
```

Comment:
Just to be consistent with the other places in the code, use [{}] in the error message, the same applies to line 675.

Reply:
Sure thing, thanks for the catch - 0c3e2f45c74873a7d92f25568199cbc745e96519